DS13/code/modules/borg/turfs.dm

```diff
@@ -0,0 +1,24 @@
+/turf/closed/wall/trek_smooth/borg
+	name = "Assimilated wall"
+	desc = "An oddly tinted wall with bits of metal, piping and green lights all over it."
+	mod = null
+	icon = 'DS13/icons/turf/trek_wall_borg.dmi'
+	icon_state = "0"
+	sheet_type = /obj/item/stack/sheet/metal
```

Comment:
Shouldn't you make a borg compound instead? Or otherwise just make it drop nothing when deconstructed.Making it drop metal sheets when you dismantle a bunch of electronics doesnt make a whole lot of sense